assets/css/style.css

```diff
@@ -0,0 +1,166 @@
+* {
```

Comment:
Use reset.css file for reseting propreties.

Reply:
oluwoleilesanmi,  The Css lint shows warnings not error.. Do you have any suggestions as to  how we can resolve them?

Reply:
- You can ignore warnings, and you may not be able to clear all.
- Just keep some of those warnings in mind, they will be helpful in writing better css later.
- Well done team :+1: